lib/components/victory-container.js

```diff
@@ -14,12 +79,14 @@ export default class extends VictoryContainer {
     const parentProps = Object.assign({className}, nativeStyle, svgProps);
     return standalone !== false ?
       (
-        <Svg {...parentProps}>
-          {title ? <title id="title">{title}</title> : null}
-          {desc ? <desc id="desc">{desc}</desc> : null}
-          {children}
-          <Portal ref={this.savePortalRef}/>
-        </Svg>
+        <View {...this.panResponder.panHandlers}>
```

Comment:
This makes it pretty definite that container components with events can not be used with custom `<svg>` at least in victory-native. I'm fine with this constraint. I'll open an issue to document it.